services/hanke-service/src/integrationTest/resources/fi/hel/haitaton/hanke/logging/expectedHankeWithPolygon.json.mustache

```diff
@@ -106,7 +106,15 @@
       {{^tormaystarkasteluTulos}}
           "tormaystarkasteluTulos": null,
       {{/tormaystarkasteluTulos}}
-      "nimi": {{#alueNimi}}"{{alueNimi}}"{{/alueNimi}}{{^alueNimi}}null{{/alueNimi}}
+      "nimi": {{#alueNimi}}"{{alueNimi}}"{{/alueNimi}}{{^alueNimi}}null{{/alueNimi}},
+      "haittojenhallintasuunnitelma": {{#haittojenhallintasuunnitelma}}{
+        "AUTOLIIKENNE":"Autoliikenteelle koituvien haittojen hallintasuunnitelma",
+        "LINJAAUTOLIIKENNE":"Linja-autoliikenteelle koituvien haittojen hallintasuunnitelma",
+        "YLEINEN":"Yleisten haittojen hallintasuunnitelma",
+        "PYORALIIKENNE":"Pyöräliikenteelle koituvien haittojen hallintasuunnitelma",
+        "RAITIOLIIKENNE":"Raitioliikenteelle koituvien haittojen hallintasuunnitelma",
+        "MUUT":"Muiden haittojen hallintasuunnitelma"
+      }{{/haittojenhallintasuunnitelma}}{{^haittojenhallintasuunnitelma}}null{{/haittojenhallintasuunnitelma}}
```

Comment:
If this is moved above the nimi row, the null-branch becomes unnecessary. Missing fields are interpreted as nulls.
```suggestion
      {{#haittojenhallintasuunnitelma}}
          "haittojenhallintasuunnitelma": {
          "AUTOLIIKENNE":"Autoliikenteelle koituvien haittojen hallintasuunnitelma",
          "LINJAAUTOLIIKENNE":"Linja-autoliikenteelle koituvien haittojen hallintasuunnitelma",
          "YLEINEN":"Yleisten haittojen hallintasuunnitelma",
          "PYORALIIKENNE":"Pyöräliikenteelle koituvien haittojen hallintasuunnitelma",
          "RAITIOLIIKENNE":"Raitioliikenteelle koituvien haittojen hallintasuunnitelma",
          "MUUT":"Muiden haittojen hallintasuunnitelma"
        },
      {{/haittojenhallintasuunnitelma}}
      "nimi": {{#alueNimi}}"{{alueNimi}}"{{/alueNimi}}{{^alueNimi}}null{{/alueNimi}}
```

The null-branch of the tormaystarkasteluTulos is also unnecessary and could be removed.

Reply:
Adjusted the template. Please note, that `tormaystarkasteluTulos` and `haittojenhallintasuunnitelma` are `boolean` values in the template object.